app/models/aid_application.rb

```diff
@@ -288,7 +288,7 @@ class AidApplication < ApplicationRecord
   belongs_to :submitter, class_name: 'User', inverse_of: :aid_applications_submitted, counter_cache: :aid_applications_submitted_count, optional: :true
   belongs_to :approver, class_name: 'User', inverse_of: :aid_applications_approved, counter_cache: :aid_applications_approved_count, optional: :true
   belongs_to :disburser, class_name: 'User', inverse_of: :aid_applications_disbursed, counter_cache: :aid_applications_disbursed_count, optional: :true
-  has_one :payment_card
+  has_one :payment_card, dependent: :destroy
```

Comment:
I'd like this to blow up actually if there is an unsubmitted application with a payment card.

Reply:
i put this one for when we delete everything on demo. but maybe we would never have a payment card on demo? i wasn't sure about that.

Reply:
Oh. I didn't even think of that. 

Actually this would be bad on demo because I want there to be seed apps on Demo. But just reset everything back to a freshly seeded state